### ISSUE TITLE
Cache github issues details

### ISSUE THREAD

**Issue by soyacz** [scylladb/argus#546]
We use user token to retrieve details for GH issues. This causes the need to ask for additional permissions upon logging in to Argus and complicates displaying issues.
E.g. copy issues modal won't show details if issue was not updated by client (which is the case for Summary View Widget and happens in View issues list for some issues sometimes).

Idea is to periodically update basic issue details (like assignee, status, tags, title) by querying GH (e.g. by fetching only latest changes or like we do in SCT/dtest in `SkipPerIssue` feature). Set up cron that will run ArgusCli command which does that.

**Comment by fruch** [comment]
I think it's would be helpful especially casuse of

**Comment by fruch** [comment]
@k0machi @soyacz, can you guys estimate this effort ?

**Comment by soyacz** [comment]
I'd say 1-2 days of work

**Comment by k0machi** [comment]
About that, yes, 1-2 days.

**Comment by fruch** [comment]
the pipeline creating the cache for SCT/dtest https://github.com/scylladb/scylla-cluster-tests/blob/master/.github/workflows/cache-issues.yaml

**Comment by soyacz** [comment]
yes, basically caching issues is done (I don't think `number,state,labels,title` should be enough).
My estimates (proposal) around what needs to be done:
1. adjust `ArgusGithubIssue` model to include labels
2. Create a systemd service that will pull every x minutes from the cache and update changed fields in `ArgusGithubIssue` table
3. drop fetching issues details with user token (and don't require full-repo-access upon login)
